venues/NeurIPS.cc/2021/Conference/python/build-conflicts.py

```diff
@@ -0,0 +1,236 @@
+import openreview
+import argparse
+from datetime import datetime
+
+
+def get_profile_info(profile):
+    
+    domains = set()
+    emails=set()
+    relations = set()
+    publications = set()
+    common_domains = ['gmail.com', 'qq.com', '126.com', '163.com', 
+                      'outloook.com', 'hotmail.com', 'yahoo.com', 'aol.com', 'msn.com', 'ymail.com', 'googlemail.com', 'live.com']
+
+    ## Institution section, get history within the last three years
+    for h in profile.content.get('history', []):
+        if h.get('end') is None or int(h.get('end')) > 2017:
+            domain = h.get('institution', {}).get('domain', '')
+            domains.update(openreview.tools.subdomains(domain))
+
+    ## Relations section, get coauthor/coworker relations within the last three years + all the other relations
+    for r in profile.content.get('relations', []):
+        if r.get('relation', '') in ['Coauthor','Coworker']:
```

Comment:
what about the relation 'advisor' or 'advisee', you won't use them as conflicts?

Reply:
We will use them as conflicts. Coauthor and coworker relations: we only want them within the last three years; all the other relations: we will add all of them as conflicts(line 26-27)

Reply:
Got it!